uwblueprint/jump-math
# feat: new question card component

## Notion ticket link
Update Question Card component (https://www.notion.so/uwblueprintexecs/Update-Question-Card-component-b33ea6f8d0d5411f96453d296abc1194)

## Implementation description
* Added the necessary icons
* Modified QuestionCard props (removed image and text; added tags, questions, points, questionNum)
* Created a QuestionTag component to display the tags accordingly 

## Steps to test
1. Go to http://localhost:3000/component-library
2. Scroll down to the question card component
3. Ensure that ellipsis show when text overflows
4. Ensure everything looks correct according to the Figma 

## What should reviewers focus on?
* if the result matches the design
* ensure that the props are correct


## Checklist
